balboa-core/src/main/scala/com/socrata/balboa/metrics/data/impl/Cassandra11Util.scala

```diff
@@ -109,28 +108,36 @@ object Cassandra11Util extends StrictLogging {
     logger.info("Setting Cassandra socket timeout to '{}'", sotimeout.toString)
     logger.info("Using keyspace '{}'", keyspace)
 
-    val connectionPoolConfiguration = new ConnectionPoolConfigurationImpl("BalboaPool")
-      .setConnectTimeout(sotimeout)
-      .setTimeoutWindow(sotimeout)
-      .setMaxConnsPerHost(connections)
-      .setSeeds(seeds)
 
     // Set local DC as side-effect if specified in configuration.
     // If the local datacenter is specified it will limit the driver
     // to only make connections to the Cassandra nodes in the datacenter
     // and prevent this service from unintentionally reaching across a
     // VPN to connect to a Cassandra node.
-    datacenter.foreach(connectionPoolConfiguration.setLocalDatacenter)
-
-    val cxt:AstyanaxContext[Keyspace] = new Builder()
-      .forKeyspace(keyspace)
-      .withAstyanaxConfiguration(new AstyanaxConfigurationImpl()
-        .setDiscoveryType (NodeDiscoveryType.RING_DESCRIBE)
-      )
-      .withConnectionPoolConfiguration(connectionPoolConfiguration)
-      .withConnectionPoolMonitor(new CountingConnectionPoolMonitor())
-      .buildKeyspace(ThriftFamilyFactory.getInstance())
-    cxt.start()
-    cxt
+
+    val dcPolicy = DCAwareRoundRobinPolicy.builder()
+    datacenter.foreach(dc => dcPolicy.withLocalDc(dc))
+
+    val poolingOptions = new PoolingOptions()
+      .setIdleTimeoutSeconds(sotimeout)
+      .setPoolTimeoutMillis(sotimeout)
```

Comment:
The order of magnitude is right here, but I don't think something called `sotimeout` = `socket timeout` is the right name for the property in the properties file. Default is probably okay here, so you can remove this line too.